checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -2025,14 +2028,19 @@ private void checkMustCall(
 
       // sometimes the store is null!  this looks like a bug in checker dataflow.
       // TODO track down and report the root-cause bug
-      CFValue aliasCFValue = cmStore != null ? cmStore.getValue(alias.reference) : null;
+      AccumulationValue aliasCFValue = cmStore != null ? cmStore.getValue(alias.reference) : null;
       AnnotationMirror cmAnno = null;
 
       if (aliasCFValue != null) { // When store contains the lhs
-        for (AnnotationMirror anno : aliasCFValue.getAnnotations()) {
-          if (AnnotationUtils.areSameByName(
-              anno, "org.checkerframework.checker.calledmethods.qual.CalledMethods")) {
-            cmAnno = anno;
+        Set<String> accumulatedValues = aliasCFValue.getAccumulatedValues();
+        if (accumulatedValues != null) {
```

Comment:
As I understand it `accumulatedValues` can only be non-null if we have a type variable or wildcard type.  While this is documented very clearly elsewhere it may be helpful to have a short comment here also:

```suggestion
        if (accumulatedValues != null) { // type variable or wildcard type
```